scripts/ansible/roles/avalanche_base/tasks/main.yml

```diff
@@ -1,15 +1,7 @@
-- name: Install Avalanche dependencies
-  become: true
-  apt:
-    name:
-      # Build
-      - cmake
-      - curl
-      - g++
-      - libssl-dev
-      - libuv1-dev
-      - make
-      # Staking key management
-      - openssl
-      - python3-cryptography
-    state: present
+- name: Dispatch tasks
+  include_tasks: "{{ item }}"
+  with_first_found:
+    - "{{ ansible_facts.distribution | lower }}-{{ ansible_facts.distribution_major_version }}.yml"
```

Comment:
Do we need to introduce some check about non-supported distributions here?

Like I run `manjaro`, and this would give me `archlinux-20.yml` which later would break of course.
However, I am not sure we need to be this diligent. But then, we are nowhere explicitly documenting which distros are supported.

Reply:
> Do we need to introduce some check about non-supported distributions here?

Good idea.

> But then, we are nowhere explicitly documenting which distros are supported.

Agreed, some documentation is on my list.